src/main/resources/application.yml

```diff
@@ -0,0 +1,29 @@
+server:
+  port:
```

Comment:
what port? What will happen if we have this field empty? Why do we need it at all if it's empty?

Reply:
If empty it means 8080.